include/fc/io/cfile.hpp

```diff
@@ -27,14 +27,16 @@ class cfile_datastream;
 class cfile {
 public:
    cfile()
-     : _file(nullptr, &fclose)
+     : _open(false),
+       _file_path(""),
+       _file(nullptr, &fclose)
    {}
 
-   void set_file_path( fc::path file_path ) {
+   void set_file_path( const fc::path &file_path ) {
       _file_path = std::move( file_path );
```

Comment:
The `std::move` here should be removed if you are going to pass by `const&`. Why was this changed?

Reply:
yes you are right. i'll fix it.

Reply:
it was an attempt to remove the function from calling the copy constructor unnecessarily. I reverted it. since the implementation a move, maybe an r-value param might be better?